data/sheragi/archaeology missions.txt

```diff
@@ -355,19 +355,47 @@ mission "Sad Archie"
 	on complete
 		log `Experienced a strange vision while drifting in the Ildaria system that looked like the fall of a civilization of dragon-people. The terrain on the planet of Zug looks strikingly similar to this vision.`
 		conversation
+			branch tmbr
+				has "Deep Archaeology 5: done"
+			action
+				set "archaeology before TMBR"
```

Comment:
Shouldn't this name be flipped? This action is skipped over if you have done the archaeology mission, so that would be TMBR before archaeology. Or is the `has` supposed to be a `not`? Looking at the rest of the changes, it seems like that's the case.